Sources/SwiftGraphQLParser/GraphQLParser.swift

```diff
@@ -52,8 +114,8 @@ public func parse(_ input: String) throws -> Document {
 			definitions.append(definition)
 		}
 	} catch let error as InternalParserError {
-		let (line, column) = error.index.lineAndColumn(in: input)
-		throw ParserError(type: error.type, line: line, column: column)
+		print(ParserError(type: error.type, errorRange: error.errorRange, input: input, contextualStart: error.contextualStart, contextualEnd: error.contextualEnd).errorDescription!)
```

Comment:
IMO, the parser itself shouldn't be responsible for printing to `stdout` or enforcing any form of error formatting. Consumers should handle catching `ParserError` and displaying it as it sees fit.

Reply:
That's an accident.  I was using it to see what Syrup displays when pointing to this branch.  I'll revert